ostd/src/arch/x86/mm/mod.rs

```diff
@@ -303,3 +304,46 @@ impl fmt::Debug for PageTableEntry {
             .finish()
     }
 }
+
+/// Enables memory-management essential features for the x86 MMU.
+pub(super) fn enable_essential_features() {
+    // PAT has been available since Pentium III (1999) and is ubiquitous in
+    // modern 64-bit CPUs. Therefore, we assume that all x86-64 CPUs should
+    // have Page Attribute Table (PAT) support. Otherwise, we should check
```

Comment:
https://github.com/asterinas/asterinas/pull/2570#discussion_r2525915985 means:
```rust
    // Page Attribute Table (PAT) has been available since Pentium III (1999)
    // and is ubiquitous in modern 64-bit CPUs. Therefore, we assume that all
    // x86-64 CPUs should have PAT support. Otherwise, we should check
```